### PR TITLE
User manage button clear status, mute meeting implementation 

### PR DESCRIPTION
**Pull request by BobakOftadeh**
This pull request implements the functionality for clearing the status of the users and muting meeting with the option to keep the presenter unmuted. #4885

### CODE REVIEW

File: bigbluebutton-html5/imports/api/voice-users/server/methods.js [hunk 1]
```diff
@@ -2,12 +2,16 @@ import { Meteor } from 'meteor/meteor';
 import mapToAcl from '/imports/startup/mapToAcl';
 import listenOnlyToggle from './methods/listenOnlyToggle';
 import muteToggle from './methods/muteToggle';
+import muteAllToggle from './methods/muteAllToggle';
+import muteAllExceptPresenterToggle from './methods/muteAllExceptPresenterToggle';
 import ejectUserFromVoice from './methods/ejectUserFromVoice';
 
 Meteor.methods(mapToAcl(['methods.listenOnlyToggle', 'methods.toggleSelfVoice',
-  'methods.toggleVoice', 'methods.ejectUserFromVoice'], {
+  'methods.toggleVoice', 'methods.ejectUserFromVoice', 'methods.muteAllUsers', 'methods.muteAllExceptPresenter'], {
```
Comment (antobinary) [inline bigbluebutton-html5/imports/api/voice-users/server/methods.js]: we removed ACL in https://github.com/bigbluebutton/bigbluebutton/pull/6226 - check it to see how to adjust pls

File: bigbluebutton-html5/imports/api/voice-users/server/modifiers/changeMuteMeeting.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    meetingId,
+  };
+
+  const modifier = {
+    $set: {
+      'voiceProp.muteOnStart': payload.muted,
+    },
+  };
+
+  return Meetings.upsert(selector, modifier);
```
Comment (KDSBrowne) [inline bigbluebutton-html5/imports/api/voice-users/server/modifiers/changeMuteMeeting.js]: `upsert` is missing a callback function for error handling.
```
  const cb = (err) => {
    if (err) {
      return Logger.error(`${err}`);
    }
    Logger.info(`some_msg`);
  };

Meetings.upsert(selector, modifier, cb);
```

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/styles.scss [hunk 3]
```diff
@@ -1,26 +1,34 @@
 @import "/imports/ui/stylesheets/variables/_all";
 
+:root{
+  --icon-offset-left: -.44em;
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/styles.scss]: Are we able to make these values more generic (less magic)?

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/styles.scss [hunk 4]
```diff
@@ -1,26 +1,34 @@
 @import "/imports/ui/stylesheets/variables/_all";
 
+:root{
+  --icon-offset-left: -.44em;
+  --icon-offset-top: -.38em;
+  --square-side-length: 1.56rem;
+  --dropdown-offset-top: 4.8rem;
```
Comment (KDSBrowne) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/styles.scss]: This as well. It's causing issue #6207.

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/container.jsx [hunk 5]
```diff
@@ -1,48 +1,83 @@
...
+  users: PropTypes.arrayOf(Object).isRequired,
+  muteAllUsers: PropTypes.func.isRequired,
+  muteAllExceptPresenter: PropTypes.func.isRequired,
+  setEmojiStatus: PropTypes.func.isRequired,
+};
+
 export default class UserOptionsContainer extends Component {
   constructor(props) {
     super(props);
+    const meetingId = Auth.meetingID;
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/container.jsx]:
``` const meetingId = Auth.meetingID;
 const meeting = Meetings.findOne({ meetingId });```
You have this in two spots. Instead you can bring it to the top as a single `const meeting = Meetings.findOne({ meetingId: Auth.meetingID });`

**Pull request merged**